agent/cmd/checkPermission.go

```diff
@@ -19,6 +19,9 @@ func init() {
 	checkPermissionCmd.Flags().String("serial-number", "", "the serial-number of the ssh certificate")
 	checkPermissionCmd.Flags().String("username", "", "the username of the user trying to authenticate")
 	checkPermissionCmd.Flags().String("key-id", "", "the key-id of the ssh certificate")
+	checkPermissionCmd.Flags().String("key-fingerprint", "", "The fingerprint's public key used at an ssh certificate.")
```

Comment:
Just to mention: you are using `.` at the end of these statements but not on the others.

Reply:
Great, dots removed.